src/sidebar/util/test/root-thread-test.js

```diff
@@ -0,0 +1,273 @@
+import * as annotationFixtures from '../../test/annotation-fixtures';
+import uiConstants from '../../ui-constants';
+import thread from '../root-thread';
+import { $imports } from '../root-thread';
+import immutable from '../immutable';
+
+const fixtures = immutable({
+  emptyThread: {
+    annotation: undefined,
+    children: [],
+  },
+  nonEmptyDraft: {
+    text: 'Some text',
+    tags: [],
+    isPrivate: false,
+  },
+});
+
+describe('sidebar/utils/rootThread', () => {
```

Comment:
```suggestion
describe('sidebar/utils/root-thread', () => {
```

The top-level `describe` block should _either_ be the name of the main export (if "sufficiently unique") or the module path.